froide_evidencecollection/importer.py

```diff
@@ -0,0 +1,508 @@
+import logging
+from collections import defaultdict
+from datetime import datetime
+
+from django.apps import apps
+from django.conf import settings
+from django.core.files.base import ContentFile
+from django.db import transaction
+from django.db.models import Q
+
+import requests
+
+from froide_evidencecollection.models import (
+    Affiliation,
+    Attachment,
+    Evidence,
+    Group,
+    Institution,
+    PersonOrOrganization,
+    Role,
+    Source,
+)
+from froide_evidencecollection.regions import selectable_regions
+from froide_evidencecollection.utils import ImportStats, get_default_value
+
+logger = logging.getLogger(__name__)
+
+CONFIG = settings.FROIDE_EVIDENCECOLLECTION_NOCODB_IMPORT_CONFIG
+API_URL = CONFIG["api_url"]
+API_TOKEN = CONFIG["api_token"]
+
+
+class TableDataFetcher:
+    def __init__(self, table_name):
+        self.table_name = table_name
+
+    def iter_rows(self):
+        offset = 0
+
+        while True:
+            data = self.fetch_from_api(offset)
+            rows = data.get("list", [])
+            page_info = data.get("pageInfo", {})
+
+            for row in rows:
+                yield row
+
+            if page_info.get("isLastPage", True):
+                break
+
+            offset += 25  # Default page size
```

Comment:
pageSize can be different, I would suggest 

```suggestion
            offset += page_info['pageSize']
```